### PR TITLE
DOMA-2717 footer spacing between company contacts

### PR DESCRIPTION
**Pull request by gegius**

### CODE REVIEW

File: apps/condo/domains/user/components/containers/PosterLayout.tsx [hunk 1]
```diff
@@ -48,8 +49,8 @@ export const PosterLayout: React.FC<IPosterLayoutProps> = ({ children, headerAct
                 <Col span={24}>
                     <Footer isSmall={isSmall}>
                         <Typography.Link
-                            href={`mailto:${SUPPORT_EMAIL}`}>{SUPPORT_EMAIL}
-                        </Typography.Link>,
+                            href={`mailto:${SUPPORT_EMAIL}`}>{SUPPORT_EMAIL_MSG}
+                        </Typography.Link>
```
[the remark this answers is not in the document]
Comment (akarjakin) [reply apps/condo/domains/user/components/containers/PosterLayout.tsx:53]: href is ok there.

[the remark this answers is not in the document]
Comment (AleX83Xpert) [reply apps/condo/domains/user/components/containers/PosterLayout.tsx:53]: See `SUPPORT_EMAIL_MSG`

[the remark this answers is not in the document]
Comment (akarjakin) [reply apps/condo/domains/user/components/containers/PosterLayout.tsx:53]: but according to UI it's better to move comma outside the link

[the remark this answers is not in the document]
Comment (gegius) [reply apps/condo/domains/user/components/containers/PosterLayout.tsx:53]: Do u think that  <Typography>{', '}</Typography>  is the best way ?

[the remark this answers is not in the document]
Comment (AleX83Xpert) [reply apps/condo/domains/user/components/containers/PosterLayout.tsx]: `{', '}` between links looks good.

**Review by AleX83Xpert — APPROVED** [review]

**Pull request merged**